src/lib/components/scheduling/SongAvailibilityTimeline.svelte

```diff
@@ -81,17 +81,15 @@
 		{/if}
 	</div>
 
-	{#if somebodyUnavailable}
-		<Column class="somebody-unavailable">Somebody is unavailable</Column>
-	{:else if timeslotPossible}
+	{#if timeslotPossible}
 		{#if minStartTime > 0}
 			<Column
 				class="red"
 				style={`max-width: ${(minStartTime / totalMinutes) * 100}% !important;`}
 			/>
 		{/if}
 		<Column
```

Comment:
Orange now shows if some people are completely unavailable, but don't you also want it to show if for a certain period 1 person is not available at that time, even though they may be later?

Reply:
Yes, I have to look into this still. For now just accept it as it is. Can't fix it in the time I have left today, so I will open another Issue for it